repos/fdbt-site/cypress_tests/cypress.config.ts

```diff
@@ -4,6 +4,7 @@ export default defineConfig({
     retries: 3,
     numTestsKeptInMemory: 0,
     redirectionLimit: 25,
+    pageLoadTimeout: 70000,
```

Comment:
Is this a 70 second timeout? How come we have added this?

Reply:
by default it is 60000 just increased it to give time for some of the tests to run as it wasn't loading the page in 60000 but the tests still pass

Reply:
I could be wrong but I feel like if the page is taking more than 60 seconds to load, there's something wrong there which waiting for 10 seconds longer won't fix. I dont think there's any harm in changing it to 70, though.

Reply:
removed it